src/dynamic-import/import-source-script-code-valid.case

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2024 Chengzhong Wu. All rights reserved.
+// Copyright (C) 2018 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: import() can be used in script code
+template: syntax/valid
+features: [source-phase-imports]
+---*/
+
+//- setup
+// This is still valid in script code, and should not be valid for module code
+// https://tc39.github.io/ecma262/#sec-scripts-static-semantics-lexicallydeclarednames
+var smoosh; function smoosh() {}
```

Comment:
What do we need this for?

Reply:
This is used to ensure that this file is been parsed as a script rather than a module. The file verifies that an `import.source()` call can be used in script code.

Reply:
This case is adapted from https://github.com/tc39/test262/blob/main/src/dynamic-import/script-code-valid.case. I believe the intention here is to disallow this file to be accidentally evaluated as a module code and pass the test as a false positive.